handlers/datasetediting/details.go

```diff
@@ -177,91 +177,91 @@ func detailsForm(l *locale.Locale, d *models.Dataset, errors validation.Errors)
 
 	identifierTypeOptions := make([]form.SelectOption, len(vocabularies.Map["dataset_identifier_types"]))
 	for i, v := range vocabularies.Map["dataset_identifier_types"] {
-		identifierTypeOptions[i].Label = l.TS("identifier", v)
+		identifierTypeOptions[i].Label = loc.Get("identifier." + v)
 		identifierTypeOptions[i].Value = v
 	}
 
 	f := form.New().
 		WithTheme("default").
-		WithErrors(localize.ValidationErrors(l, errors)).
+		WithErrors(localize.ValidationErrors(loc, errors)).
 		AddSection(
 			&form.Text{
 				Name:     "title",
 				Value:    d.Title,
-				Label:    l.T("builder.title"),
+				Label:    loc.Get("builder.title"),
 				Cols:     9,
-				Error:    localize.ValidationErrorAt(l, errors, "/title"),
+				Error:    localize.ValidationErrorAt(loc, errors, "/title"),
 				Required: true,
 			},
 			&form.Select{
 				Name:        "identifier_type",
 				Value:       identifierType,
-				Label:       l.T("builder.identifier_type"),
+				Label:       loc.Get("builder.identifier_type"),
 				Options:     identifierTypeOptions,
 				Cols:        3,
-				Help:        template.HTML(l.T("builder.identifier_type.help")),
-				Error:       localize.ValidationErrorAt(l, errors, "/identifier"),
+				Help:        template.HTML(loc.Get("builder.identifier_type.help")),
+				Error:       localize.ValidationErrorAt(loc, errors, "/identifier"),
 				EmptyOption: true,
 				Required:    true,
 			},
 			&form.Text{
 				Name:     "identifier",
 				Value:    identifier,
 				Required: true,
-				Label:    l.T("builder.identifier"),
+				Label:    loc.Get("builder.identifier"),
 				Cols:     3,
-				Help:     template.HTML(l.T("builder.identifier.help")),
-				Error:    localize.ValidationErrorAt(l, errors, "/identifier"),
-				Tooltip:  l.T("tooltip.dataset.identifier"),
+				Help:     template.HTML(loc.Get("builder.identifier.help")),
+				Error:    localize.ValidationErrorAt(loc, errors, "/identifier"),
+				Tooltip:  loc.Get("tooltip.dataset.identifier"),
 			},
 		).
 		AddSection(
 			&form.SelectRepeat{
 				Name:        "language",
-				Label:       l.T("builder.language"),
-				Options:     localize.LanguageSelectOptions(l),
+				Label:       loc.Get("builder.language"),
+				Options:     localize.LanguageSelectOptions(),
```

Comment:
localize.LanguageSelectOption(loc) => loc arg ontbreekt hier?

Reply:
nee is niet meer nodig